### PR TITLE
Changed rentEpoch type in AccountHeader from number to BigInt

### PR DESCRIPTION
**Pull request by HarishPrasannaV**
Addressed issue #99 
- I ran unit tests, and it worked. (please don't get mad if I messed up, I'm still a noob)

### CODE REVIEW

File: packages/umi-rpc-web3js/src/createWeb3JsRpc.ts [hunk 1]
```diff
@@ -384,7 +394,7 @@ function parseAccount(
     executable: account.executable,
     owner: fromWeb3JsPublicKey(account.owner),
     lamports: lamports(account.lamports),
-    rentEpoch: account.rentEpoch,
+    rentEpoch: contvertToBigInt(account.rentEpoch),
```
Comment (lorisleiva) [inline packages/umi-rpc-web3js/src/createWeb3JsRpc.ts:397]: What about this instead?

```suggestion
    rentEpoch: account.rentEpoch ? BigInt(account.rentEpoch) : undefined,
```

Comment (HarishPrasannaV) [reply packages/umi-rpc-web3js/src/createWeb3JsRpc.ts]: Thanks for the suggestion, I just implemented it.

**Comment by HarishPrasannaV** [comment]
Umm...I don't understand why it failed the first test.
It passed on the first commit??????

**Comment by HarishPrasannaV** [comment]
INPUT_BRANCH value: issue_99_fix
Should I merge the branch with main and then give pull request again?

**Comment by lorisleiva** [comment]
It's a linting problem. If you run `pnpn lint:fix && pnpn format:fix` then commit the result, it should be happy.

**Review by lorisleiva — APPROVED** [review]
Thanks!

**Pull request merged**